src/server/scripts/Outland/HellfireCitadel/ShatteredHalls/boss_nethekurse.cpp

```diff
@@ -264,8 +270,47 @@ class spell_tsh_shadow_sear : public SpellScriptLoader
         }
 };
 
+class spell_tsh_shadow_bolt : public SpellScriptLoader
+{
+    public:
+        spell_tsh_shadow_bolt() : SpellScriptLoader("spell_tsh_shadow_bolt") { }
+
+        class spell_tsh_shadow_bolt_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_tsh_shadow_bolt_SpellScript);
+
+            void SelectRandomPlayer(WorldObject*& target)
+            {
+                if (Creature* caster = GetCaster()->ToCreature())
+                {
+                    std::list<Player*> playerList;
+                    playerList.clear();
```

Comment:
why clear here?

Reply:
I don't know, took this over from here:
https://github.com/azerothcore/azerothcore-wotlk/blob/e22d78ecd6997bc11f8620b759364f119dac431a/src/server/scripts/Northrend/AzjolNerub/ahnkahet/instance_ahnkahet.cpp#L268-L269
Is it not necessary?

Reply:
not to me
@GMKyle

Reply:
Just read that it is not necessary, as the default constructor of std::list automatically creates an empty list. I pushed a new commit removing the line.